.babelrc

```diff
@@ -0,0 +1,16 @@
+{
+    "presets": [
+        "@babel/preset-env",
+        "@babel/preset-flow"
+    ],
+    "plugins": [
+        "@babel/plugin-transform-runtime",
+        [
+            "module-resolver",
+            {
+                "root": ["src", "."],
+                "extensions": [".js", ".json"]
+            }
+        ]
+    ]
+}
```

Comment:
From our discussion, we can specify the target output environment in the babel config file. This should now compile code to be used with the version of node currently running by the machine at compile time. From what I found, I don't think we can restrict the input versions of ECMAScript though since Babel doesn't seem to differentiate between them.